cylc/flow/scripts/install.py

```diff
@@ -56,12 +56,12 @@
   # run directory ~/cylc-run/fido/run1
   $ cylc install --flow-name=fido
 
-  # Install $PWD/bunny/rabbit/flow.cylc as "bunny/rabbit", with run directory
-  # ~/cylc-run/bunny/rabbit/run1
+  # Install $PWD/bunny/rabbit/flow.cylc as "rabbit", with run directory
+  # ~/cylc-run/rabbit/run1
   $ cylc install --directory=bunny/rabbit
```

Comment:
Naïve test
```bash
> cd ~/cylc-src
> mkdir -p bunny/rabbit
> touch bunny/rabbit/flow.cylc
> cylc install --directory=bunny/rabbit
INSTALLED cylc-src from bunny/rabbit -> /home/h02/tpilling/cylc-run/**cylc-src/run1**
> cylc install --directory=bunny/rabbit
```

Reply:
Hmm, that was supposed to be fixed by #4161, I'll look into it

Reply:
Cannot reproduce, I get
```
INSTALLED rabbit from bunny/rabbit -> ~/cylc-run/rabbit/run1
```

Reply:
Same, perhaps you hadn't checked out this branch @wxtim?